op-dispute-mon/mon/resolutions.go

```diff
@@ -52,7 +52,7 @@ func (r *ResolutionMonitor) CheckResolutions(games []*types.EnrichedGameData) {
 	for _, game := range games {
 		complete := game.Status != gameTypes.GameStatusInProgress
 		duration := uint64(r.clock.Now().Unix()) - game.Timestamp
-		maxDurationReached := duration >= game.Duration
+		maxDurationReached := duration >= game.MaxClockDuration
```

Comment:
If clock extension triggers for this game, then the game would no longer be `complete`.

Reply:
yeah this one might cause some false alarms because games appear to not have resolved in time due to clock extension. I'm tempted to just leave it as-is though because if we're hitting cases where clock extension is required, something funky is going on and we probably should take a look anyway.
The only way I could see to counter it is to have the contract record any time extensions it grants which seems excessive.  We don't want to check the remaining duration on clocks here because we want to detect if someone finds a way to extend the clocks beyond what should be possible.